src/components/portfolio/VolatilityLineEntry.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Tooltip } from '@material-ui/core';
+
+type VolatilityLineEntryProps = {
+  volatilityValue: number;
+  tooltipText: string;
+  color: string;
+};
+
+const VolatilityLineEntry: React.FC<VolatilityLineEntryProps> = ({
+  volatilityValue,
+  tooltipText,
+  color,
+}) => {
+  const useStyles = makeStyles({
+    line: {
+      position: 'absolute',
+      height: '2rem',
+      width: '0.25rem',
+      transform: 'translate(0%, -50%)',
+    },
+    tooltip: {
+      fontSize: '0.8rem',
+      fontWeight: 600,
+      lineHeight: '1.2rem',
+      whiteSpace: 'pre-line',
+    },
+  });
+
+  const classes = useStyles();
+
+  return (
+    <Tooltip title={<p className={classes.tooltip}>{tooltipText}</p>}>
```

Comment:
I don't think paragraph is needed here, can't you just give Tooltip that class? If not I'd make a div around it.

Reply:
I followed the implementation/styling of Julian's analyst bar component, which implemented the styling in this way. Not sure how to keep it consistent otherwise

Reply:
Nevermind, played around with the tags a bit and managed to fix it. I'll commit this right after